conan/tools/files/files.py

```diff
@@ -549,3 +532,17 @@ def move_folder_contents(conanfile, src_folder, dst_folder):
         os.rmdir(src_folder)
     except OSError:
         pass
+
+
+class FileProgress(io.FileIO):
```

Comment:
This file is not the right place for this class, this file should be intended only for recipe importable ``from conan.tools``. Things from this module shouldn't be used by Conan internals, and this needs to be imported from the internal "upload" code.

Reply:
Okey, totally right, I've moved back to previous `conans` location. Consider moving in the future to `conan.internal`